### ISSUE TITLE
Travis-CI test always fails.

### ISSUE THREAD

**Issue by sushichop** [CocoaLumberjack/CocoaLumberjack#806]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [x] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | iOS / tvOS
 Platform Version        | iOS 10.0 / tvOS 9.0
 CocoaLumberjack Version | 3.0.0
 Integration Method      | travis-ci
 Xcode Version           | Xcode 8.0 / 8.1
 Repro rate              | all the time (100%)
 Demo project link       | N/A

### Issue Description and Steps

Travis-CI test always fails. Error log is like this.

```
** TEST FAILED **
The command "xcodebuild test -workspace Framework/Lumberjack.xcworkspace -scheme 'iOS Tests' -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 6,OS=latest' | xcpretty -c" exited with 65.
```

I dug it and I have finally solved this issue. 
So I'll send PR to solve this issue later.

**Comment by sushichop** [comment]
PR to solve this issue is #807.

**Comment by sushichop** [comment]
Fixed by #807 .
